app/code/Magento/Wishlist/view/frontend/layout/wishlist_index_index.xml

```diff
@@ -7,12 +7,16 @@
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <update handle="customer_account"/>
+    <head>
```

Comment:
This block is not needed, styles will be merged and rendered to the page by the framework

Reply:
ok